kidney_transplant_llm/pydantic_study_variables.py

```diff
@@ -19,9 +36,14 @@ class SpanAugmentedMention(BaseModel):
 
 # Dates are treated as strings - no enum needed
 class DonorTransplantDateMention(SpanAugmentedMention):
+    """
+    Date of the first renal transplant. If there is only one transplant mentioned, 
+    assume that this is the first transplant. If a POD (post-operative day) is mentioned, 
+    use that to infer the date.
+    """
     donor_transplant_date: str | None = Field(
         None,
-        description='Exact date of renal transplant; use YYYY-MM-DD format in your response. Only highlight date mentions with an explicit day, month, and year (e.g. 2020-01-15). All other date mentions, or an absence of a date mention, should be indicated with None.'
+        description='Exact date of first renal transplant; use YYYY-MM-DD format in your response. Only highlight date mentions with an explicit day, month, and year (e.g. 2020-01-15). All other date mentions, or an absence of a date mention, should be indicated with None.'
```

Comment:
I tacitly approve, I often see just "month and day" syntax especially for the expected  scenario where the true transplant date is near the coded  `sample_casedef_index_date`. Nonetheless I see value in this strict formalism as it removes guesswork.